js/src/Ice/ConnectionI.js

```diff
@@ -561,17 +556,13 @@ export class ConnectionI {
                     return;
                 }
 
-                this._transceiver.unregister();
```

Comment:
I guess this was redundant, setting the holding state does the same.

Reply:
Yes, and it's even more obvious now that setState(StateHolding) is called only once.